src/TextForEvent.js

```diff
@@ -17,6 +17,13 @@ limitations under the License.
 var MatrixClientPeg = require("./MatrixClientPeg");
 var CallHandler = require("./CallHandler");
 
+const roles = {
+    undefined: 'Default',
+    0: 'User',
+    50: 'Moderator',
+    100: 'Admin',
+};
+
```

Comment:
It feels like the sort of thing that should be in one place: we should probably reuse the logic from/in https://github.com/matrix-org/matrix-react-sdk/blob/master/src/components/views/elements/PowerSelector.js